tests/geometryRTheta/advection_rtheta/advection_simulation_utils.hpp

```diff
@@ -49,7 +49,7 @@ std::string to_lower(std::string s)
  * @param[in] to_physical_mapping
  *      The mapping function from the logical domain to the physical domain.
  * @param[in] idx_range_theta
- *      The index range to which the poloidal coordinate should be restricted.
+ *      The index range of the domain to which the poloidal coordinate should be restricted.
```

Comment:
```suggestion
 *      An index range spanning the domain to which the poloidal coordinate should be restricted.
```
?
Each domain could have multiple index ranges so it is not clear which one "The index range" refers to